docs/site/Include-filter.md

```diff
@@ -0,0 +1,240 @@
+---
+title: 'Include filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, include
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Include-filter.html
+---
+
+An _include_ filter enables you to include results from related models in a query over relations. (See [Relations](Relations.md) for details on how to define relations)
+
+The value of the include filter can be a string, an array, or an object.
+
+{% include important.html content="You can use an _include_ filter with `find(),` `findOne()` and `findById()` methods.
+" %}
+
+### Node.js API
+
+{% include content/angular-methods-caveat.html lang=page.lang %}
+
+To query one relation:
+```ts
+{include: [{relation: 'relationName'}]}
+```
+
+To query multiple relations:
+```ts
+{include: [{relation: 'relationName1'}, {relation: 'relationName2'}]}
+```
+
+To query nested relations, use the scope field:
+
+```ts
+{
+    relation: 'relationName',
+    scope: {
+    include: [{relation: 'nestedRelationName'}],
+    },
+}
+```
+
+Where:
+
+- _relationName_ is the name of a relation defined in repositories. Check [Relations](Relations.md) for details.
+
+### REST API
+
+To query one relation:
+`/modelName?filter[include][][relation]=_relationName_`
```

Comment:
`[]` empty?

Reply:
Yes, when there is only one relation, `[]` is good. And to use `[0], [1]`... for multiple relations.